### ISSUE TITLE
Keyboard shortcuts broken in GTK+ dialogs

### ISSUE THREAD

**Issue by ckerr** [electron/electron#19755]
Introduced in PR #19725 to address issue #19663 

That change drops the underscores from the strings passed to the GTK dialog, which breaks the keyboard shortcuts for those dialog buttons.

See this similar [Chromium discussion](https://cs.chromium.org/chromium/src/chrome/browser/ui/libgtkui/select_file_dialog_impl_gtk.cc?type=cs&q=SelectFileDialogImplGTK::CreateFileOpenHelper&sq=package:chromium&g=0&l=40):

```c++
#if GTK_CHECK_VERSION(3, 90, 0)
// GTK stock items have been deprecated.  The docs say to switch to using the
// strings "_Open", etc.  However this breaks i18n.  We could supply our own
// internationalized strings, but the "_" in these strings is significant: it's
// the keyboard shortcut to select these actions.  TODO(thomasanderson): Provide
// internationalized strings when GTK provides support for it.
const char kCancelLabel[] = "_Cancel";
const char kOpenLabel[] = "_Open";
const char kSaveLabel[] = "_Save";
#else
G_GNUC_BEGIN_IGNORE_DEPRECATIONS
const char* const kCancelLabel = GTK_STOCK_CANCEL;
const char* const kOpenLabel = GTK_STOCK_OPEN;
const char* const kSaveLabel = GTK_STOCK_SAVE;
G_GNUC_END_IGNORE_DEPRECATIONS
#endif
```

tl;dr the underscores are important.

Also relevant to the previous PR, Chromium appears to not suffer from the same translation issues. I don't know why that is but it may be worth investigating. Chromium [uses](https://cs.chromium.org/chromium/src/chrome/browser/ui/libgtkui/select_file_dialog_impl_gtk.cc?type=cs&q=SelectFileDialogImplGTK::CreateFileOpenHelper&sq=package:chromium&g=0&l=327) those `kFooLabel`s from above this way:

```c++
GtkWidget* SelectFileDialogImplGTK::CreateFileOpenHelper(
    const std::string& title,
    const base::FilePath& default_path,
    gfx::NativeWindow parent) {
  GtkWidget* dialog = gtk_file_chooser_dialog_new(
      title.c_str(), nullptr, GTK_FILE_CHOOSER_ACTION_OPEN, kCancelLabel,
      GTK_RESPONSE_CANCEL, kOpenLabel, GTK_RESPONSE_ACCEPT, nullptr);
  SetGtkTransientForAura(dialog, parent);
  AddFilters(GTK_FILE_CHOOSER(dialog));
```

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.
